src/aspire/source/relion.py

```diff
@@ -106,10 +105,12 @@ def __init__(
             "_rlnAmplitudeContrast",
         ]
         # If these all exist in the STAR file, we may create CTF filters for the source
-        if set(CTF_params).issubset(metadata.columns):
+        if set(CTF_params).issubset(metadata.keys()):
             # partition particles according to unique CTF parameters
             filter_params, filter_indices = np.unique(
```

Comment:
Again, maybe split this up into two lines.

Reply:
refactored.